src/commands/transfer.ts

```diff
@@ -0,0 +1,52 @@
+import type yargs from "yargs";
+import type { Arguments, CommandBuilder } from "yargs";
+import { GlobalOptions } from "../cli.js";
+import { setupCommand } from "../lib/commandSetup.js";
+
+export interface Options extends GlobalOptions {
+  name: string;
+  receiver: string;
+}
+
+export const command = "transfer <name> <receiver>";
+export const desc = "Transfer a table to another address";
+
+export const builder: CommandBuilder<{}, Options> = (yargs) =>
+  yargs
+    .positional("name", {
+      type: "string",
+      description: "The target table name",
+    })
+    .positional("receiver", {
+      type: "string",
+      description: "The address to transfer the table to",
+    }) as yargs.Argv<Options>;
+
+export const handler = async (argv: Arguments<Options>): Promise<void> => {
+  try {
+    const { name, receiver, chain } = argv;
+    const parts = name.split("_").reverse();
+    const chainId = parts[1];
```

Comment:
If the table is named `my_table_of_dog_names_31337_123` will this still work?
Can we use the wasm-sqlparser for this?

Reply:
Idk if this is any useful, but I did something very similar in the Discord bot. I used the wasm parser to get table names and then parsed that for the chain/table IDs (lines 20-26): https://github.com/tablelandnetwork/tableland-bot/blob/main/commands/read.js

Mine is a little simplistic and doesn't really handle multiple tables, but the idea is there. Tbh, that would be a useful method in the parser itself (getParts->return prefix, chainId, tableId).

Reply:
I _think_ you can use the sqlparser's validateTableName method.  
The README has an example, but I'm not sure if the README is up to date.
https://github.com/tablelandnetwork/wasm-sqlparser#usage

Reply:
Nice! I used the parser instead of the more simplistic function. It _would_ have worked for your table name, but I like deferring to the parser. Thanks